atomicapp/nulecule/container.py

```diff
@@ -122,12 +122,18 @@ def extract_files(self, image, source, dest):
                   '%s:/%s' % (container_id, source), dest]
         logger.debug(
             'Copying data from docker container: %s' % ' '.join(cp_cmd))
-        subprocess.check_output(cp_cmd)
+        try:
+            subprocess.check_output(cp_cmd, stderr=subprocess.STDOUT)
+        except subprocess.CalledProcessError as e:
+            raise DockerException('Copying data from docker container Failed! %s. \n%s' % (cp_cmd, e.output))
```

Comment:
To clarify:

Remove `'Copying data from docker container Failed! ` to simply say `'Copying data from docker container failed:`